packages/web/config/chain-infos.ts

```diff
@@ -3242,6 +3242,34 @@ const chainInfos = (
       features: ["ibc-transfer", "ibc-go"],
       explorerUrlToTx: "https://www.mintscan.io/omniflix/txs/{txHash}",
     },
+    {
+      rpc: "https://a.client.sentry.net.bluzelle.com:26657",
+      rest: "https://a.client.sentry.net.bluzelle.com:1317",
+      chainId: "bluzelle-9",
+      chainName: "Bluzelle",
+      bip44: {
+        coinType: 483,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("bluzelle"),
+      currencies: [
+        {
+          coinDenom: "BLZ",
+          coinMinimalDenom: "ubnt",
+          coinDecimals: 6,
+          coinImageUrl: "/tokens/bluzelle.svg",
+          coinGeckoId: "bluzelle",
```

Comment:
please define the price using a pool's spot price (will require that a pool be created)